src/BaGet/appsettings.json

```diff
@@ -1,5 +1,6 @@
 ﻿{
   "ApiKeyHash": "",
+  "PathBase": "",
```

Comment:
Let's remove the `PathBase` option out of the default `appsettings.json` file. I'd like to keep the default `appsettings.json` file as simple as possible as to not overwhelm new users.